set.mm

```diff
@@ -66901,6 +66978,18 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
       ( wcel cfv wceq fvmptg syl2anc ) ACFMEHMCGNEOKLBCDEFHGJIPQ $.
   $}
 
+  ${
+    $d ph x $.  $d A x $.  $d C x $.  $d D x $.
+    fvmptd4.1 $e |- ( x = A -> B = C ) $.
+    fvmptd4.2 $e |- ( ph -> F = ( x e. D |-> B ) ) $.
+    fvmptd4.3 $e |- ( ph -> A e. D ) $.
+    fvmptd4.4 $e |- ( ph -> C e. V ) $.
+    $( Deduction version of ~ fvmpt (where the substitution hypothesis does not
+       have the antecedent ` ph ` ).  (Contributed by SN, 26-Jul-2024.) $)
+    fvmptd4 $p |- ( ph -> ( F ` A ) = C ) $=
```

Comment:
optional: fvmptd4 shortens fvmptd3

Reply:
The shorter proof uses more definitions (~df-sbc, ~df-csb), but the same axioms, so the shortening is valid.

Reply:
Hm, after the proof was shortened using ~fvmptd4, a DV condition must be added ( $d ph x $. ), and this would cause a lot of DV violations for proofs using ~fvmptd3. Therefore, I will not change this proof.